### PR TITLE
Add Traditional Chinese support

### PR DESCRIPTION
**Pull request by haeky**
Since `zh.yml` is only Simplified Chinese and most Taiwainese uses Traditional Chinese, I went ahead and added it to `zh-TW.yml`.

### CODE REVIEW

File: rails/locale/iso_639-1/zh-TW.yml [hunk 1]
```diff
@@ -1,3 +1,263 @@
...
+    TJ：塔吉克斯坦
+    TK：托克勞
+    TL：東帝汶
+    TM：土庫曼斯坦
+    TN：突尼斯
+    TO：湯加
+    TR：土耳其
+    TT：特立尼達和多巴哥
+    TV：圖瓦盧
+    TW：台灣
```
Comment (jlungc) [inline rails/locale/iso_639-1/zh-TW.yml]: :+1:

File: rails/locale/iso_639-1/zh-TW.yml [hunk 2]
```diff
@@ -1,3 +1,263 @@
...
+    GN：幾內亞
+    GP：瓜德羅普島
+    GQ：赤道幾內亞
+    GR：希臘
+    GS：南喬治亞與南桑威奇群島
+    GT：危地馬拉
+    GU：關島
+    GW：幾內亞比紹
+    GY：圭亞那
+    HK：中國香港特別行政區
```
Comment (jlungc) [inline rails/locale/iso_639-1/zh-TW.yml]: Some HK's might be offended by calling HK a district of China :grimacing:

File: rails/locale/iso_639-1/zh-TW.yml [hunk 3]
```diff
@@ -1,3 +1,263 @@
...
+    JO：約旦
+    JP：日本
+    KE：肯尼亞
+    KG：吉爾吉斯斯坦
+    KH：柬埔寨
+    KI：基里巴斯
+    KM：科摩羅
+    KN：聖基茨和尼維斯
+    KP：朝鮮
+    KR：韓國
```
Comment (jlungc) [inline rails/locale/iso_639-1/zh-TW.yml]: Not sure if you want to differentiate North and South Korea -> 北韓 (north) and 南韓 (south).

File: rails/locale/iso_639-1/zh-TW.yml (hunk 1 above)
Comment (jlungc) [inline rails/locale/iso_639-1/zh-TW.yml]: Can also use 臺灣.

**Comment by haeky** [comment]
Any idea why it got removed in c4eb95a397949c77ee57c0e10b0a681b52ca6588?

**Comment by haeky** [comment]
I've added `zh-HK` but it's the same as `zh-TW` to prevent it to fallback to `zh`. I'm not sure if `zh-CN` should just fallback to Simplified Chinese? 

Thoughts @HangingClowns?

**Comment by haeky** [comment]
ping @onomojo

**Comment by onomojo** [comment]
Thanks for this

**Pull request merged**